test/e2e_node/device_manager_test.go

```diff
@@ -287,7 +327,7 @@ var _ = SIGDescribe("Device Manager  [Serial] [Feature:DeviceManager][NodeFeatur
 			}, time.Minute, time.Second).Should(gomega.BeTrue())
 
 			ginkgo.By("Scheduling a sample device plugin pod")
-			data, err := e2etestfiles.Read(SampleDevicePluginDS2YAML)
+			data, err := e2etestfiles.Read(SampleDevicePluginControlRegistrationDSYAML)
```

Comment:
this manifest correctly uses the 1.5 plugin version, which is not promoted yet. This is just fine for now. Let's promote the plugin and we can move forward.

Reply:
Thanks for getting the image promoted (https://github.com/kubernetes/k8s.io/pull/4862). I'll run a final check very shortly and review (and likely LGTM) again. I see no issues at all, just the last bit of due diligence.